capstone-frontend/src/components/ClubSearchList.jsx

```diff
@@ -15,60 +15,60 @@ export class ClubSearchList extends Component {
 
   getData = async () => {
     // Get clubs by searchQuery
-    let clubs = await fetch(`/api/clubSearch?searchTerm=${this.props.searchQuery}`)
+    let clubs = await fetch(`/api/clubs?searchTerm=${this.props.searchQuery}`)
```

Comment:
nit: Can be `const` now that you're not reassigning

Reply:
With the new concat instead of push, I have to reassign it with `clubs = clubs.concat(clubsWithBook)` since it does not seem to update the array itself without reassignment.